bigbluebutton-html5/package.json

```diff
@@ -6,7 +6,8 @@
     "preinstall": "npm install grunt-cli",
     "test": "cd app;JASMINE_BROWSER=PhantomJS JASMINE_MIRROR_PORT=3000 meteor run --test",
```

Comment:
There's only one `test` command. The file you are pointing to is 15 commits behind. Look at the history of the file: https://github.com/MaximKhlobystov/bigbluebutton/commits/41645c315cc7bf7f5e027887bf1204869ec4d31c/bigbluebutton-html5/package.json.
This is the actual file on the branch: https://github.com/MaximKhlobystov/bigbluebutton/blob/8ac038e6a1f8fa08b71db59cd21a17744ef152e5/bigbluebutton-html5/package.json.